src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.ToJson.Enumerable.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This method should definitely not be marked with AI. I doubt it helps.

Reply:
It is only called once, but often, so a candidate for inlining.

Reply:
I doubt the JIT can inline it though, even if you ask. cc @jkotas, @AndyAyersMS